src/v2/BaseLoginRouter.js

```diff
@@ -224,7 +225,12 @@ export default Router.extend({
     // state token (which will be set into AppState)
     if (this.settings.get('oieEnabled')) {
       try {
-        const idxResp = await startLoginFlow(this.settings);
+        let idxResp = await startLoginFlow(this.settings);
+        /* eslint-disable max-depth */
+        if (!this.appState.get('hasRendered')) {
```

Comment:
proactively we should check here if there is any configured flow needs to be handled..?